### PR TITLE
Home Screen Unit Tests

### PR DESCRIPTION
**Pull request by oscar-amezquita**
## Description
Add Unit test classes: 
- Add BasePresenter Unit test (Test attach and detach state, register / unregister bus).
- Add ListUsersPresenterImp Unit test. (Test get user list success, get users list without view attached).

### CODE REVIEW

File: app/src/test/java/com/globant/example/mentorapp/com/globant/example/mentorapp/home/presentation/presenter/ListUsersPresenterImplTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+public class ListUsersPresenterImplTest {
+
+    @Mock
+    FetchUserListInteractor interactor;
+    @Mock
+    BaseView view;
+    @InjectMocks
+    ListUsersPresenterImpl presenter;
+
+
```
Comment (daniellemariani) [inline app/src/test/java/com/globant/example/mentorapp/com/globant/example/mentorapp/home/presentation/presenter/ListUsersPresenterImplTest.java]: Remove extra line

File: app/src/test/java/com/globant/example/mentorapp/mvp/base/BasePresenterTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+public class BasePresenterTest {
+
+    @Mock
+    private BaseView view;
+    @InjectMocks
+    private BasePresenter presenter;
+    @Mock
+    private Bus bus;
+
+
```
Comment (daniellemariani) [inline app/src/test/java/com/globant/example/mentorapp/mvp/base/BasePresenterTest.java]: Remove extra line

**Review by daniellemariani — COMMENTED** [review]
LG, just minors comments

File: app/src/test/java/com/globant/example/mentorapp/com/globant/example/mentorapp/home/presentation/presenter/ListUsersPresenterImplTest.java (hunk 1 above)
Comment (oscar-amezquita) [reply app/src/test/java/com/globant/example/mentorapp/com/globant/example/mentorapp/home/presentation/presenter/ListUsersPresenterImplTest.java]: Done.

File: app/src/test/java/com/globant/example/mentorapp/mvp/base/BasePresenterTest.java (hunk 2 above)
Comment (oscar-amezquita) [reply app/src/test/java/com/globant/example/mentorapp/mvp/base/BasePresenterTest.java]: Done

**Review by daniellemariani — APPROVED** [review]
LG

**Pull request merged**